atlp-rwanda/cabal-phantom-frontend
# #WYfBlp40 Setup continuous integration and Heroku Pipeline

#### What does this PR do?
Requests to merge Ch-frontend-setup-continuous-integration-and-testing-WYfBlp40 into develop

#### Description of Task to be completed?

- Setup Heroku pipeline

- Configured Testing environment with Jest & Enzyme

- Travis CI is configured as a continuous integration service

- Travis CI badge has been added to your repo

- Configure CodeClimate for code analysis and Maintainability

#### How should this be manually tested?
-clone the repo and checkout origin/Ch-frontend-setup-continuous-integration-and-testing-WYfBlp40 

- run `npm install` 

- run `npm test`

- Go to [Heroku](https://cabal-fronte-ch-fronten-w7str4.herokuapp.com/) to see deployment

- check the travis and codeclimate badges  on This [Branch](https://github.com/atlp-rwanda/cabal-phantom-frontend/tree/Ch-frontend-setup-continuous-integration-and-testing-WYfBlp40)


#### What are the relevant pivotal tracker stories?
[WYfBlp40](https://trello.com/c/WYfBlp40/27-setup-continuous-integration-quality-assurance-pipeline)

#### Screenshots (if appropriate)
<img width="1440" alt="Screen Shot 2020-11-24 at 10 57 49" src="https://user-images.githubusercontent.com/68019992/100071726-0b780180-2e44-11eb-87be-772c67d99fff.png">
